extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -193,6 +200,21 @@ class AmpCarousel extends AMP.BaseElement {
     // Need to wait for slides to exist first.
     this.carousel_.goToSlide(Number(this.element.getAttribute('slide') || '0'));
     // Signal for runtime to check children for layout.
+
+    if (
+      isAmp4Email(this.element.ownerDocument) ||
```

Comment:
`amp-carousel:0.2` is not allowed in email yet, so this check is not necessary.

Reply:
Ah! that explains, I was wonder why we didn't have the check here : )